docs/services/rook-services-guide.md

```diff
@@ -0,0 +1,652 @@
+---
+title: Using Rook Services
+toc: true
+weight: 450
+indent: true
+---
+# Deploying Yugastore with Rook
+
+This user guide will walk you through [Yugastore] application deployment using
+Crossplane's [Rook] stack to run [YugabyteDB] in a Google Cloud [GKE] Kubernetes
+cluster. To do so, we will broadly:
+
+1. Provision a GKE Kubernetes cluster
+1. Install the Rook [Yugabyte operator] into the GKE cluster
+1. Provision a YugabyteDB cluster and deploy the Yugastore app into the GKE cluster
+
```

Comment:
I added these broad steps at the beginning because it this guide feels like a pretty different use-case than the other services guide. Might be worth adding a note at the top saying *Advanced Guide* or something...